pom.xml

```diff
@@ -253,6 +250,22 @@
 					<artifactId>maven-compiler-plugin</artifactId>
 					<version>3.6.1</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<!-- newer versions seem to not show stacktraces in Netbeans -->
+					<version>2.18</version>
+				</plugin>
+				<plugin>
+					<groupId>io.github.zlika</groupId>
+					<artifactId>reproducible-build-maven-plugin</artifactId>
+					<version>0.7</version>
```

Comment:
Is this used at more than 1 place?

Reply:
@deathbeam yes it is used in the runelite-client module. However a version was never set in that module previously.
<img width="912" alt="Screen Shot 2019-06-01 at 8 23 54 AM" src="https://user-images.githubusercontent.com/16943514/58749732-c6d5fa80-8446-11e9-91e2-78411dae2572.png">

Reply:
Ah alright